docs/interpreter/spark.md

```diff
@@ -190,29 +191,24 @@ Once `SPARK_HOME` is set in `conf/zeppelin-env.sh`, Zeppelin uses `spark-submit`
   <tr>
     <th>spark-defaults.conf</th>
     <th>SPARK_SUBMIT_OPTIONS</th>
-    <th>Applicable Interpreter</th>
```

Comment:
AFAIK, there settings are applicable for all the spark interpreters. So I remove this column.